src/ensdf_processing/HSICC/bldhst.for

```diff
@@ -0,0 +1,131 @@
+C***********************************************************************01 00010
+C*                                                                      01 00020
```

Comment:
Yes, we have permission.
On Jan 27, 2016 12:05, "Josh[ua] Howland" <notifications@github.com> wrote:

> In src/ensdf_processing/HSICC/bldhst.for
> <https://github.com/pyne/pyne/pull/809#discussion_r51014710>:
>
> > @@ -0,0 +1,131 @@
> > +C***********************************************************************01 00010
> > +C*                                                                      01 00020
>
> @rachelslaybaugh <https://github.com/rachelslaybaugh> we think we do have
> permission, correct?
> @scopatz <https://github.com/scopatz> common extension? The extension
> (.f, .for, etc) varies, I left it in the form it was distributed with.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyne/pyne/pull/809/files#r51014710>.
>